app/src/main/java/org/mozilla/focus/widget/FirefoxProgressBar.kt

```diff
@@ -35,9 +35,11 @@ class FirefoxProgressBar @JvmOverloads constructor(
         browserFrag.session.loading.observe(browserFrag, object : NonNullObserver<Boolean>() {
             public override fun onValueChanged(loading: Boolean) {
                 if (loading) {
+                    visibility = View.VISIBLE
                     setMaxVisibility()
                 } else {
                     resetCountdown()
+                    visibility = View.GONE
```

Comment:
This is going to hide the view before the animation runs, right? Do we still want the animation?

Reply:
To follow up, this should be using an animation listener.